samples/python/django/sample.py

```diff
@@ -0,0 +1,145 @@
+import os
+import sys
+import datetime
+import django
+import pytz
+from django.db import connection
+from django.db.transaction import atomic
+from django.db import transaction
+x = 0
+
+def get_singer(singer_id):
```

Comment:
nit: Could we rename these `get_` methods to something like `create_test_singer` etc.? `get_singer` sounds like reading the singer from the database.

Reply:
done